configs/hydradx.yml

```diff
@@ -1,22 +1,99 @@
 endpoint: wss://hydradx-rpc.dwellir.com
 mock-signature-host: true
 block: ${env.HYDRADX_BLOCK_NUMBER}
-db: ./db.sqlite
+db: ./hydradx.db.sqlite
 # wasm-override: hydradx_runtime.compact.compressed.wasm
 
 import-storage:
-  System:
```

Comment:
no need for system::account?

Reply:
Interlay uses Tokens for both native and external assets.

Reply:
Thanks, this is needed. Added it back.